umm-spec-lib/src/cmr/umm_spec/umm_to_xml_mappings/iso_smap.clj

```diff
@@ -190,7 +168,7 @@
             (for [instrument (distinct (mapcat :Instruments (:Platforms c)))]
               [:gmd:keyword
                (char-string (kws/smap-keyword-str instrument))])]]
-          (generate-user-constraints c)
+          (use-constraints/generate-user-constraints c)
```

Comment:
iso-smap and iso19115's generate-user-constraints are different, which was why I didn't put it in the iso_shared. Looks like your use-constraints/generate-user-constraints is the same as the iso19115's?  Am I missing sth?

Reply:
Other than the base path the mappings appear identical.

If you were to start from MI_Metadata the paths are equal:

Here are the full paths for UseConstraints:
ISO19115-2:
**/gmi:MI_Metadata/gmd:identificationInfo/gmd:MD_DataIdentification/gmd:resourceConstraints/gmd:MD_LegalConstraints/gmd:useLimitation/gco:CharacterString**

SMAP:
/gmd:DS_Series/gmd:seriesMetadata<b>/gmi:MI_Metadata/gmd:identificationInfo/gmd:MD_DataIdentification/gmd:resourceConstraints/gmd:MD_LegalConstraints/gmd:useLimitation/gco:CharacterString</b>

AccessConstraints/Description:
ISO19115-2:
**/gmi:MI_Metadata/gmd:identificationInfo/gmd:MD_DataIdentification/gmd:resourceConstraints/gmd:MD_LegalConstraints/gmd:useLimitation/gco:CharacterString** (prefix: 'Restriction Comment:' )

SMAP:
/gmd:DS_Series/gmd:seriesMetadata<b>/gmi:MI_Metadata/gmd:identificationInfo/gmd:MD_DataIdentification/gmd:resourceConstraints/gmd:MD_LegalConstraints/gmd:useLimitation/gco:CharacterString</b> (prefix: 'Restriction Comment:' )

AccessConstraints/Value:
ISO19115-2:
**/gmi:MI_Metadata/gmd:identificationInfo/gmd:MD_DataIdentification/gmd:resourceConstraints/gmd:MD_LegalConstraints/gmd:otherConstraints/gco:CharacterString** (prefix: 'Restriction Flag:' )

SMAP:
/gmd:DS_Series/gmd:seriesMetadata<b>/gmi:MI_Metadata/gmd:identificationInfo/gmd:MD_DataIdentification/gmd:resourceConstraints/gmd:MD_LegalConstraints/gmd:otherConstraints/gco:CharacterString</b> (prefix: 'Restriction Flag:' )

I retrieved these values from the UMM mappings excel sheet, hopefully I got that right!